src/fsharp/PrettyNaming.fs

```diff
@@ -293,7 +417,7 @@ module internal Microsoft.FSharp.Compiler.PrettyNaming
             res <- res && n.[i] >= '0' && n.[i] <= '9';
         res
 
-    type NameArityPair = NameArityPair of string*int
+    type NameArityPair = NameArityPair of name:string * arity:int
```

Comment:
Named DU fields is a 3.1 feature, but our internal build uses a pre-3.1 compiler to bootstrap.  So for now we can't do this.